README.md

```diff
@@ -1 +1,1042 @@
+
+* **3.1.15** If you have a function that takes no arguments, has no side effects, and returns some object or value, prefer using a computed property instead.
+
+* **3.1.16** For the purpose of namespacing a set of `static` functions and/or `static` properties, prefer using a caseless `enum` over a `class` or a `struct`. This way, you don't have to add a `private init() { }` to the container.
+
+### 3.2 Access Modifiers
+
+* **3.2.1** Write the access modifier keyword first if it is needed.
+
+```swift
+// PREFERRED
+private static let myPrivateNumber: Int
+
+// NOT PREFERRED
+static private let myPrivateNumber: Int
+```
+
+* **3.2.2** The access modifier keyword should not be on a line by itself - keep it inline with what it is describing.
+
+```swift
+// PREFERRED
+open class Pirate {
+    /* ... */
+}
+
+// NOT PREFERRED
+open
+class Pirate {
+    /* ... */
+}
+```
+
+* **3.2.3** In general, do not write the `internal` access modifier keyword since it is the default.
+
+* **3.2.4** If a property needs to be accessed by unit tests, you will have to make it `internal` to use `@testable import ModuleName`. If a property *should* be private, but you declare it to be `internal` for the purposes of unit testing, make sure you add an appropriate bit of documentation commenting that explains this. You can make use of the `- warning:` markup syntax for clarity as shown below.
+
+```swift
+/**
+ This property defines the pirate's name.
+ - warning: Not `private` for `@testable`.
+ */
+let pirateName = "LeChuck"
+```
+
+* **3.2.5** Prefer `private` to `fileprivate` where possible.
+
+* **3.2.6** When choosing between `public` and `open`, prefer `open` if you intend for something to be subclassable outside of a given module and `public` otherwise. Note that anything `internal` and above can be subclassed in tests by using `@testable import`, so this shouldn't be a reason to use `open`. In general, lean towards being a bit more liberal with using `open` when it comes to libraries, but a bit more conservative when it comes to modules in a codebase such as an app where it is easy to change things in multiple modules simultaneously.
+
+### 3.3 Custom Operators
+
+Prefer creating named functions to custom operators.
+
+If you want to introduce a custom operator, make sure that you have a *very* good reason why you want to introduce a new operator into global scope as opposed to using some other construct.
+
+You can override existing operators to support new types (especially `==`). However, your new definitions must preserve the semantics of the operator. For example, `==` must always test equality and return a boolean.
+
+### 3.4 Switch Statements and `enum`s
+
+* **3.4.1** When using a switch statement that has a finite set of possibilities (`enum`), do *NOT* include a `default` case. Instead, place unused cases at the bottom and use the `break` keyword to prevent execution.
```

Comment:
Porém fica explícito que você não quis usar alguns caso e não que esqueceu saca? E se algum cara for adicionado você vai ser obrigado implementar algo, nem que seja mais um com break nesses lugares.